src/algorithms/rhythm/tempocnn.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2006-2020  Music Technology Group - Universitat Pompeu Fabra
+ *
+ * This file is part of Essentia
+ *
+ * Essentia is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation (FSF), either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the Affero GNU General Public License
+ * version 3 along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+
+#include "tempocnn.h"
+
+using namespace std;
+using namespace essentia;
+using namespace standard;
+
+const char* TempoCNN::name = "TempoCNN";
+const char* TempoCNN::category = "Rhythm";
+const char* TempoCNN::description = DOC(
+  "This algorithm makes predictions using Hendrik Schreiber's TempoCNN-based models.\n"
```

Comment:
The first sentence (short description) of TempoCNN is equivalent to TensorflowPredictTempoCNN. They should be distinguished somehow. 

TensorflowPredictTempoCNN predicts from a given input mel-spectrogram, while TempoCNN predicts from a given input audio.